### PR TITLE
Added Uzbek language

### PR DESCRIPTION
**Pull request by ogabek96**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ ] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ ] Make sure you are merging your commits to `dev` branch.
* [ ] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: src/locale/lang/uz-UZ.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+exports.__esModule = true;
+exports.default = {
```
Comment (island205) [inline src/locale/lang/uz-UZ.js]: These lines above this one are not need.

File: src/locale/lang/uz-UZ.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    select: {
+      loading: 'Yuklanmoqda',
+      noMatch: 'Mos ma`lumot yo`q',
+      noData: 'Ma`lumot yo`q',
+      placeholder: 'Tanladizngiz'
+    },
+    cascader: {
+      noMatch: 'Mos ma`lumot yo`q',
+      loading: 'Yuklanmoqda',
+      placeholder: 'Tanladingiz'
```
Comment (island205) [inline src/locale/lang/uz-UZ.js]: Miss line https://github.com/ElemeFE/element/blob/dev/src/locale/lang/en.js#L71

File: src/locale/lang/uz-UZ.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      emptyText: 'Ma`lumot yo`q'
+    },
+    transfer: {
+      noMatch: 'Mos ma`lumot topilmadi',
+      noData: 'Ma`lumot yo`q',
+      titles: ['List 1', 'List 2'],
+      filterPlaceholder: 'Kalit so`zni kiriting',
+      noCheckedFormat: '{total} ta element',
+      hasCheckedFormat: '{checked}/{total} belgilandi'
+    }
```
Comment (island205) [inline src/locale/lang/uz-UZ.js]: Miss https://github.com/ElemeFE/element/blob/dev/src/locale/lang/en.js#L109-L114

**Comment by island205** [comment]
@ogabek96 please add the missing parts

**Comment by ogabek96** [comment]
> @ogabek96 please add the missing parts

Fixed.Please look.

**Pull request merged**